release/release_logs/1.9.0/scalability/object_store.json

```diff
@@ -0,0 +1,10 @@
+{
+  "broadcast_time": 74.81429146999994,
```

Comment:
It's way faster for some reasons... @iycheng did you run it with the master commit branch? (that object store doesn't scale up)?

Reply:
Yes, I'm using the master commit branch for testing since some tests are only fixed in master.

Reply:
I believe the timing on this test is _extremely_ noisy, so as long as it doesn't crash, I think this is fine. The shuffle tests are a much much better benchmark for detecting performance regressions I think.

Reply:
In the master, I separated out the node startup logic from the test. Maybe that’s why